test/unit/org/apache/cassandra/utils/CassandraGenerators.java

```diff
@@ -76,6 +81,11 @@
     private static final Gen<Integer> SMALL_POSITIVE_SIZE_GEN = SourceDSL.integers().between(1, 30);
     private static final Gen<Boolean> BOOLEAN_GEN = SourceDSL.booleans().all();
 
+    public static final Gen<InetAddressAndPort> INET_ADDRESS_AND_PORT_GEN = rnd -> {
+        InetAddress address = Generators.INET_ADDRESS_GEN.generate(rnd);
+        return InetAddressAndPort.getByAddressOverrideDefaults(address, SMALL_POSITIVE_SIZE_GEN.generate(rnd));
```

Comment:
it would be better to use a port generator which returns values between `0-65,535`